### ISSUE TITLE
Pictures now have watermarks...

### ISSUE THREAD

**Issue by adamclerk** [RandomAPI/Randomuser.me-Data#50]
Why have the user pictures changed?

**Comment by xitude** [comment]
+1

**Comment by julien-c** [comment]
If bandwidth cost is an issue, could you please upload the pictures to this repo?

**Comment by xitude** [comment]
@julien-c I don't think you understand what a repo is for, its for codebase not file storage.

**Comment by CristianMoisei** [comment]
Who is going to use pictures of people that look like they have tattoos on their faces.
Seriously, get rid of the watermark. I haven't used the product since you added those.

**Comment by SeanHub** [comment]
Why the watermark? Doesn't look anywhere near as good.

**Comment by julien-c** [comment]
Adding watermarked to creativecommons-sourced pictures from somebody else is also in incredibly poor taste.

**Comment by CristianMoisei** [comment]
Will you please remove those damn watermarks so we can get back to using the product?
